.github/CODEOWNERS

```
@@ -0,0 +1,6 @@
+# Added platform team as code owners so that PRs do not get missed
+Dockerfile @transcom/truss-platform-team
```

What do you think about adding @transcom/truss-is3 or @transcom/truss-sr-eng to this as well? This will at least spread out the ownership of these files a bit more so folks don't get stuck only being able to ping the Platform team. 

```suggestion
Dockerfile @transcom/truss-platform-team @transcom/truss-is3 @transcom/truss-sr-eng
```